cypress/e2e/supervision/clients/edit-client.cy.js

```diff
@@ -9,15 +9,17 @@ const editClient = (isCourtReferenceChanged) => {
       method: "PUT",
       url: `/supervision-api/v1/clients/${id}`,
     }).as("editClientCall");
+    cy.reload();
     cy.get('input[name="firstName"]').should('be.visible');
     cy.get('input[name="firstName"]').should("have.value", firstname);
-    cy.get('input[name="firstName"]').should("not.be.disabled");
+    cy.get('input[name="firstName"]', { timeout: 30000 }).should("not.be.disabled");
```

Comment:
The default timeout in Cypress is 60000. Is there a reason why setting these to half that improves the test reliability?

Reply:
I'm not sure why it worked but it did seem to help.